azure/examples/cloud-ts-thumbnailer/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "video-thumbnailer",
+    "version": "0.1.0",
+    "main": "index.js",
+    "scripts": {
+        "build": "tsc"
+    },
+    "dependencies": {
+        "@pulumi/pulumi": "^0.15.1-dev",
+        "@pulumi/azure": "^0.15.0",
+        "@pulumi/azure-serverless": "^0.15.0",
+        "azure-arm-containerinstance": "^3.0.0",
+        "ms-rest-azure": "^2.5.7"
```

Comment:
@lukehoban Interesting situation here.  `@pulumi/cloud` needs these two references so that at *runtime* it can load the azure sdk and then do things like spawn off a container instance.

because we filter out our @pulumi node_modules, we dont' end up including this reference, and so actually using cloud.Task doesn't work at runtime.

Adding hte references directly here solves hte issue, but is effectively a mine for anyone who comes along and wants to use the cloud-azure API.

--

We don't have this problem with AWS because aws-sdk is always available in AWS Lambdas.  So, even if cloud-aws needs the aws-sdk, it's there at runtime.  

--

One approach we could take would be to add these azure APIs we depend on at runtime to the list of 'required packages' we add when computing the FunctionApp zip file.